dashboard/app/models/lesson.rb

```diff
@@ -270,6 +270,9 @@ def summarize(include_bonus_levels = false, for_edit: false)
       if has_lesson_plan
         lesson_data[:lesson_plan_html_url] = lesson_plan_html_url
         lesson_data[:lesson_plan_pdf_url] = lesson_plan_pdf_url
+        if script.include_student_lesson_plans
```

Comment:
I think we probably also want to check that the script is_migrated here just to be safe. I know we are preventing setting this for non-migrated scripts already so its just extra safety